README.md

```diff
@@ -10,18 +10,79 @@ Requirements
 ------------
 
 * Django 1.6
-* django-localflavor
-* django-databrowse
 
 
 Installation
 ------------
 
+1. Install from GitHub.
+    ```sh
+        $ pip install git+git://github.com/unt-libraries/django-accession.git
+    ```
+
+2. Add app and dependency to INSTALLED_APPS.
+    ```python
+        INSTALLED_APPS = (
+            'django_databrowse',
+            'accession',
+        )
+    ```
+
+3. Include the URLs.
+    ```python
+        urlpatterns = [
+            url(r'^admin/', include(admin.site.urls)),
+            url(r'^tfc/', include('accession.urls')),
+        ]
+    ```
+
+4. Sync the database.
+    ```sh
+        $ python manage.py syncdb
+    ```
+
+
+Developing
+----------
+
+1. [Install Docker](http://docs.docker.com/installation/).
+
+2. Install Docker Compose.
+    ```sh
+        $ pip install docker-compose
+    ```
+
+3. Clone the repository.
+    ```sh
+        $ git clone https://github.com/unt-libraries/django-accession
+    ```
+
+4. Warm-up the database. It will take 10-15 seconds to complete after it has been pulled from the Docker Hub.
+    ```sh
+        $ docker-compose up -d db
+    ```
+
+5. Sync the database.
+    ```sh
+        $ docker-compose run --rm web python manage.py syncdb
+    ```
+
+6. Start the containers as daemons.
+    ```sh
+        $ docker-compose up -d
+        # Use 'docker-compose stop' to stop the containers.
+    ```
+    At this point you should be able to access your local instance of the site by visiting <dockerhost>:8000/tfc/databrowse.
```

Comment:
Might want to take a look at the rendered version of the README here, @somexpert